packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -811,6 +811,18 @@ describe('TransactionController', () => {
       );
     });
 
+    it('checks pending transactions', () => {
```

Comment:
Could we describe what the controller does when it checks pending transactions? E.g., it 'starts polling if there are pending transactions'.